### PR TITLE
Update uk.m3u

### LINKED ISSUE #7285: Add: Blaze TV
### Channel Name

Blaze TV

### Channel ID

_No response_

### Website

https://watch.blaze.tv/live/

### Stream URL

https://live.blaze.tv/blaze_live.m3u8?key=6Pr3Uj2Fo3Ix7Rb9Ze9Ro4Ez2Eq7Sy&platform=chrome&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9saXZlLmJsYXplLnR2L2JsYXplX2xpdmUubTN1OD9rZXk9NlByM1VqMkZvM0l4N1JiOVplOVJvNEV6MkVxN1N5JnBsYXRmb3JtPWNocm9tZSIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTY1MjQ3MzE0Nn0sIklwQWRkcmVzcyI6eyJBV1M6U291cmNlSXAiOiIyYTAyOmM3Zjo1YTQ2OjUyMDA6NTUwNjpiYzE1OjcyZjQ6ZGQyMS8zMiJ9fX1dfQ__&Expires=1652473146&Signature=NonFH-d5KTzqZYYlAwAjYNqHRpkhnt2qKmttVw9c89-0VDM7ypKC9zzTL9EZtVo3ZfID2ZUs5NiJuruhwVPkstVj16jYmaHg5X5p8~FMYXB2vG9g9ZtQj7G6qBFVwwgivryKsPdXQsC3NNKILyFS8UrXccdMQ66B8jHJIRKwZddkEHDHUofhMFkAMwgayYtycZ9J8TCi7W8kRN8c4qPz-iajFi6oqYS5~l10t3XUcW72NNnx7~T6rTYi79LZGQsTFn9-EWSc21wX0K055qRgQsfNa6tO0xLh~QZU-XRYLddHWAZ6Rrf5ADoGNPSjIJ4pztXiR5pruZVIKenQ5KMJbw__&Key-Pair-Id=APKAIRYDR7ZRJY4BSCYA

### Notes

This is a channel from the UK which airs documentaries and general entertainment programmes. You will most probably need to use lanesh4d0w5's proxy. The stream is tokened with a key, a policy and a pair id of some sort. Not sure how it's generated. Tried to use his proxy url thingy to generate it but nothing. Maybe it's generated a different way. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by UltraHDR**
closes #7285

### CODE REVIEW

**Review by its0x08 — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Host uktv.vercel.app returns 404.

File: streams/uk.m3u [hunk 1]
```diff
@@ -185,6 +187,14 @@ http://ott-cdn.ucom.am/s37/index.m3u8
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/ddrricky/live
 #EXTINF:-1 tvg-id="FadakTV.uk" status="error",Fadak TV (1080p) [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCBgM-sKkB4ySrdiCsk0ikUA/live
+#EXTINF:-1 tvg-id="Channel5.uk" status="error",Channel 5 (720p)
+https://uktv.vercel.app/five.m3u
```
Comment (LaneSh4d0w) [inline streams/uk.m3u:191]:
```suggestion
https://uktv.vercel.app/five.m3u8
```

File: streams/uk.m3u [hunk 2]
```diff
@@ -185,6 +187,14 @@ http://ott-cdn.ucom.am/s37/index.m3u8
...
+#EXTINF:-1 tvg-id="" status="error",Channel 5 USA (720p)
+https://uktv.vercel.app/fiveusa.m3u
```
Comment (LaneSh4d0w) [inline streams/uk.m3u:193]:
```suggestion
https://uktv.vercel.app/fiveusa.m3u8
```

File: streams/uk.m3u [hunk 3]
```diff
@@ -142,6 +142,8 @@ http://103.199.161.254/Content/bbcworld/Live/Channel(BBCworld)/index.m3u8
 #EXTINF:-1 tvg-id="BBCWorldNewsEurope.uk" status="online" user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148",BBC World News (480p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s24/index.m3u8
+#EXTINF:-1 tvg-id="BlazeUK.uk" status="online",Blaze UK (720p)
+https://uktv.vercel.app/blaze.m3u8
```
Comment (LaneSh4d0w) [inline streams/uk.m3u:146]: 404

File: streams/uk.m3u (hunk 2 above)
Comment (UltraHDR) [reply streams/uk.m3u:193]: Thanks

File: streams/uk.m3u (hunk 3 above)
Comment (UltraHDR) [reply streams/uk.m3u:146]: Geo-blocked maybe

**Review by benevenstanciano — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**